docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+services:
+  proxy:
+    image: mrcide/beebop-proxy:${GIT_SHA}
+    depends_on:
+      - beebop-server
+    ports:
+      - 80:80
+      - 443:443
+    command:
+      $HOST
+  beebop-server:
+    image: mrcide/beebop-server:${GIT_SHA}
+    depends_on:
+      - beebop-py-api
+  beebop-py-api:
+    image: mrcide/beebop-py:main
+    depends_on:
+      - beebop-redis
+    volumes:
+      - beebop-storage:/beebop/storage
+    environment:
+      - REDIS_HOST=beebop-redis
+      - STORAGE_LOCATION=./storage
+      - DB_LOCATION=./storage/GPS_v4_references
+  beebop-redis:
+    image: redis:5.0
+  beebop-py-worker:
+    image: mrcide/beebop-py:main
+    depends_on:
+      - beebop-redis
+    command:
+      rqworker
+    volumes:
+      - beebop-storage:/beebop/storage
+    environment:
+      - REDIS_HOST=beebop-redis
+volumes:
+  beebop-storage:
```

Comment:
Is this volume empty? When trying out the deployed app the backend analyses won't run. What I suspect is that the database is maybe missing in the volume (We previously [downloaded the db](https://github.com/bacpop/beebop/blob/e485e7625e45d187ad9d00d308b0063ee49e67fa/scripts/run_test#L12) into the volume with [this script](https://github.com/bacpop/beebop_py/blob/da6bc1f11181c47c7a64343b09cd35c1f8695410/scripts/download_db))? 
Is there a place where the logs of beebop-py-api can be viewed?

Reply:
I figured a bunch of this would be wrong, I just blindly copied this from the existing local run script! Perhaps we can have a call so you can explain how the db/redis works :)